fighthealthinsurance/fax_utils.py

```
@@ -501,15 +504,21 @@ def _create_async_ssh_client(self):
         if path[0] == "/":
-            target = f"/tmp/{self.username}{path}"
+            target = f"{self.remote_scratch_dir}/{self.username}{path}"
         try:
             sftp_client = ssh.open_sftp()
-            # Make the remote directory if needed.
+            # Make the remote directory if needed. umask 007: both fax
+            # identities (root Temporal worker, ray actor) share this tree, so
+            # dirs must be group-writable (the scratch root is setgid fax) --
+            # with the default 022 whichever identity mkdirs first locks the
+            # other out, and PII dirs shouldn't be world-readable anyway.
             dir = os.path.dirname(target)
```

Ruff (0.15.21)

[error] Variable `dir` is shadowing a Python builtin

(A001)